birdhouse/README.rst

```diff
@@ -262,6 +262,11 @@ To run the tests:
     python3 -m pip install -r tests/requirements.txt
     pytest tests/
 
+Some tests require internet access (to access JSON schemas used to validate
+JSON structure). If you need to run tests offline, you can skip the tests that
+require internet access by setting the `DACCS_SKIP_ONLINE_TESTS` environment
+variable to a non-empty value.
```

Comment:
Instead of using custom logic, it would be much better to take advantage of pytest markers and add something like `@pytest.mark.online` to the relevant tests. It is easier after to ignore using `pytest -k 'not online' tests`, which allows any logical combination of markers as needed. It will also automatically perform the skip, no need to handle it in try/expect and `pytest.fail` re-raise.